src/TestingUtils/Microsoft.AspNetCore.Testing/src/HelixQueues.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace Microsoft.AspNetCore.Testing
+{
+    public static class HelixQueues
+    {
+        public const string All = "all";
+        public const string None = "none";
+
+        // Queue names end in ';' because it makes it easier to concat these into a list using a constant expression:
+        //  HelixQueues.Fedora28 + HelixQueues.Centos7
+
+        public const string Fedora28 = "Fedora.28." + HelixSuffix + ";";
+        public const string Fedora27 = "Fedora.27." + HelixSuffix + ";";
+        public const string Redhat7 = "Redhat.7." + HelixSuffix + ";";
+        public const string Debian9 = "Debian.9." + HelixSuffix + ";";
+        public const string Debian8 = "Debian.8." + HelixSuffix + ";";
+        public const string Centos7 = "Centos.7." + HelixSuffix + ";";
+        public const string Ubuntu1604 = "Ubuntu.1604." + HelixSuffix + ";";
+        public const string Ubuntu1810 = "Ubuntu.1810." + HelixSuffix + ";";
+        public const string macOS1012 = "OSX.1012." + HelixSuffix + ";";
+        public const string Windows10 = "Windows.10.Amd64.ClientRS4.VS2017.Open;"; // Doesn't have the default suffix!
+
+        private const string HelixSuffix = "Amd64.Open";
```

Comment:
Unfortunately we did/will have Arm64 queues too, they just were exploding so they aren't ready for frontline action yet.  Maybe we need a Flaky Queue too :)

Reply:
I don't think we need a flaky queue, with this logic we can have our test scripts run both flaky **and** non-flaky tests, but only fail the build on non-flaky failures.

We can definitely update this as we have new queue explosions :). I can put `Amd64` in the field names for now to ensure they don't break when we add `Arm64` though.

Reply:
Yeah what I mean is its possible we will want to run on new queues (i.e. arm testing) where the queues themselves have a high failure rate, today we just removed the queue entirely, but it would be nice to be able to run all tests on a flaky queue (just in the don't fail the build bucket).  We would have to mark every test with `[Flaky(OnHelixQueues="imsoflaky")]` I think without the concept.  Not something we need now, but something to keep in mind for the future